From ad161e98700b31e262db3f9ac2dd4e9761602faa Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Ekstr=C3=B6m?= <jeebjp@gmail.com>
Date: Sun, 17 Feb 2019 23:13:00 +0200
Subject: [PATCH] decoder: add curly braces around a multi-line statement

It seems like these are meant to be executed together when correct
data is decoded. Right now it would always return one in this
default case.
---
 src/decoder.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/decoder.c b/src/decoder.c
index 2b77f38..69ea351 100644
--- a/src/decoder.c
+++ b/src/decoder.c
@@ -1140,8 +1140,10 @@ static int decoder_handle_time( arib_decoder_t *decoder )
                 break;
             default:
                 if( i_mode == 1 && c >= 0x40 && c <= 0x7F )
+                {
                     decoder->i_control_time += c & 0x3f;
                     return 1;
+                }
                 return 0;
         }
         if( i_mode == 0 )
